astropy/io/fits/connect.py

```diff
@@ -89,6 +90,20 @@ def read_table_fits(input, hdu=None, astropy_native=False):
     astropy_native : bool, optional
         Read in FITS columns as native astropy objects where possible instead
         of standard Table Column objects. Default is False.
+    memmap : bool, optional
+        Whether to use memory mapping, which accesses data on disk as needed. If
+        you are only accessing part of the data, this is often more efficient.
+        If you want to access all the values in the table, and you are able to
+        fit the table in memory, you may be better off leaving memory mapping
```

Comment:
I'm not sure about this one. I'm asking because I'm curious about the answer, and because the FITS FAQ tells a different story (http://docs.astropy.org/en/latest/io/fits/appendix/faq.html#what-performance-differences-are-there-between-astropy-io-fits-and-fitsio):
>  In the case of reading over all the data in the file the performance difference between using mmap versus reading the entire data into physical memory at once can vary widely between systems, hardware, and depending on what else is happening on the system at the moment, but mmap almost always going to be better.

Reply:
If you need *all* the data, then not using mmap is faster, both with images and tables, for sure. If you need only a small part of the data then mmap is definitely better. In between, it's a gray area. I think that FAQ item is misleading because for cases like reading a table and doing Table.info('stats') then you definitely get better performance without mmap.

Reply:
From my tests (on datacubes, not tables), when loading all the data, it is faster with memmap. It may depend on the OS and other things, I don't know.